src/js/popup.js

```diff
@@ -333,6 +346,26 @@ $(document).ready(function () {
         });
     });
 
+    $('#rate').click(function () {
+        dmUIClick('rate');
+        chrome.runtime.sendMessage({
+            JDTIfunction: 'setRateStatus'
+        }, function (response) {
+            if (response.status === 'success') {
+                // Flag set
+                window.open('https://chrome.google.com/webstore/detail/jira-template-injector/hmhpegjieopgbdmpocdmfkafjgcdmhha/reviews?hl=en', '_blank'); // Open extension user reviews page
+            } else {
+                if (response.message) {
+                    dmError('export', response.message);
```

Comment:
I can't remember off the top of my head but I believe the first variable into dmError is the name of the error. So it wouldn't be 'export' it would be something like 'rate'